packages/server/postgres/queries/generated/getIntegrationProvidersByIdsQuery.ts

```diff
@@ -27,6 +27,7 @@ export interface IGetIntegrationProvidersByIdsQueryResult {
   clientSecret: string | null;
   serverBaseUrl: string | null;
   webhookUrl: string | null;
+  privateKey: string | null;
```

Comment:
@Dschoordsch this was deprecated in favor of consumerSecret, right?
should we drop the column in the newest migration?